src/m365/spo/commands/user/user-get.ts

```diff
@@ -76,7 +76,10 @@ class SpoUserGetCommand extends SpoCommand {
   }
 
   #initOptionSets(): void {
-    this.optionSets.push({ options: ['id', 'email', 'userName'] });
+    this.optionSets.push({
+      options: ['id', 'email', 'userName'],
+      runsWhen: (args) => args.options.id || args.options.userName || args.options.email
+    });
   }
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
```

Comment:
Noticed that the verbose message below this line is a bit odd. Could we fix that one as well please?